All Core Devs Meetings/meeting 94.md

```diff
@@ -0,0 +1,437 @@
+**93.2.2**: Vitalik to start coordinating a writeup on the impact of the gas limit increase (what will it break)
+**93.2.3**: revisit the discussion on EIP 384
+  
+  
+## Helpful Links
+Link | Description
+--|--
+**94.3.1**: [EIP 2666 analysis results](https://github.com/ethereum/pm/issues/200#issuecomment-678115862)
+**93.3.2**:  [Context on 1884](https://github.com/holiman/eip-1884-security/)
+  
+---
+  
+# Agenda
+  
+<!-- MDTOC maxdepth:6 firsth1:1 numbering:0 bullets:1 updateOnSave:1 -->
+- [1. Progress on EIP-1057](#-1)
+- [2. Ewasm Team: Findings on "evm384"](#-2)
+- 3 EIP Discussion
+    - [3a. Vitalik's Idea](#3)
+- [4. YOLO / YOLOv2 & Berlin state tests update](#4)
+- [5. Raw benchmarks for EIP 2666](#5)
+- [6. EIPIP Meeting: New EIP Statuses](#6)
+- [7. Ethereum Cat Herders Survey Results](#7)
+- [8. Migration from ACD Gitter to Eth R&D Discord](#8)
+- [9. Review previous decisions made and action items (if notes available)](#9)
+  
+---
+  
+# 1 Progress on EIP-1057
+  
+Video | [3:56](https://youtu.be/q6bIeSu7r9o?t=236)
+  
+**Hudson** - Hello and welcome to a theorem poor developer meeting No. 94, I'm your host, Hudson, and we're going to go to the first item on the agenda. Greg wants to give an update on the progress of EIP 10 57. So, Greg, if you want to admit and go through that real quick, that'd be great.
+  
+**Greg** - Yeah, we've been reviewing it. Oh, gosh. Since about April. We had a small suggestion from. Beast authority on the light evaluation attack and kick found a and exploit in.. there's actually a bug in its hash that crockpot W made just barely maybe exploitable in some future time. But within a few days we had a fix and by about May the fix was pretty well reviewed and tweaked so that it didn't have a performance impact. And soon enough I'll be pointing to a better reference implementation of the the latest for for the WIPA. And it's it's very readable right now. There's some stuff to do, maybe patching up links and things like that, so at the next meeting, I want to actually take that up way back. In meeting EIGHTY-ONE, we published a decision to move forward with it, in 82 we backed off and it was unclear, but the overall consensus was to go with with bans, compromise of just getting it up on tests, nets and keeping it ready, but not making an immediate decision on whether to deploy it. Just keep on it, keep an eye on the state of the network. So two weeks from now is when I want to discuss that. And by the end of the weekend, I'll put up some links to two places to do that.
+  
+**Hudson** - great. I do want to ask I'm not seeing in the decisions made section in the notes anything about us definitely spinning up testnets for prog pow or spinning up Ropstein or anything specific like that. Did we in your recollection, was there like a specific tests that we talked about spinning up for it, or was it Ropstein or what?
```

Comment:
great. I do want to ask I'm not seeing in the decisions made a section in the notes anything about us definitely spinning up testnets for prog pow or spinning up Ropsten or anything specific like that. Did we in your recollection, was there like a specific tests that we talked about spinning up for it, or was it Ropsten or what?